source/comply/compliance-export.rst

```diff
@@ -26,13 +26,14 @@ Enterprise deployments with a requirement to archive history beyond the data ret
 Set up guide
 ------------
 
-Use the following guides to configure exports for CSV, Actiance XML, or Global Relay EML. \
-
-For self-hosted deployments, compliance exports are written to the ``exports`` subdirectory of the configured `Local Storage directory </configure/configuration-settings.html>`__ in the chosen format. If you've configured Mattermost to use S3 storage, the exports are written to the ``exports`` directory in the Mattermost bucket.
+Use the following guides to configure exports for CSV, Actiance XML, or Global Relay EML.
 
 .. note::
    
-   The compliance exports do not contain posts sent before the feature was enabled. For self-hosted deployments, you can export past history via the ``export`` :doc:`command line tool <../manage/command-line-tools>`. 
+   - For self-hosted deployments, compliance exports are written to the ``exports`` subdirectory of the configured `Local Storage directory </configure/environment-configuration-settings.html#file-storage>`__ in the chosen format.
+   - If you've configured Mattermost to use S3 storage, the exports are written to the ``exports`` directory in the Mattermost bucket.
+   - Alternatively, you can specify an alternate filestore target and generate an S3 presigned URL for compliance exports. See the `dedicated export filestore target </configure/experimental-configuration-settings.html#enable-dedicated-export-filestore-target>`__ configuration settings documentation for details.
```

Comment:
One little suggestion here.

```suggestion
   - For self-hosted deployments, compliance exports are written to the ``exports`` subdirectory of the configured filestore in the chosen format. This will either be in the `Local Storage directory </configure/environment-configuration-settings.html#file-storage>` or the Mattermost S3 bucket if S3 storage is configured.
   - Alternatively, you can specify an alternate filestore target and generate an S3 presigned URL for compliance exports. See the `dedicated export filestore target </configure/experimental-configuration-settings.html#enable-dedicated-export-filestore-target>`__ configuration settings documentation for details.
```

Please improve this if needed of course, but my intention is to help clarify that the dedicated export will override all standard compliance export locations.